packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -5,87 +5,48 @@ import colors from '@celo/react-components/styles/colors'
+import { earn1, earn2, earn3, earnMain } from 'src/images/Images'
 import { noHeader } from 'src/navigator/Headers'
 import { navigate, navigateBack } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
 import useSelector from 'src/redux/useSelector'
-import { stableTokenBalanceSelector } from 'src/stableToken/reducer'
-import { getContentForCurrentLang } from 'src/utils/contentTranslations'
-import Logger from 'src/utils/Logger'
-import { formatFeedDate, getNextMondayAt8pmUtc } from 'src/utils/time'
+import { accountAddressSelector } from 'src/web3/selectors'
 
 const TAG = 'ConsumerIncentivesHomeScreen'
 
-const useConsumerIncentivesContent = () => {
-  const contentResult = useAsync<ConsumerIncentivesData>(fetchConsumerRewardsContent, [])
-  let texts
-  if (contentResult.result) {
-    texts = getContentForCurrentLang(contentResult.result.content)
-  }
-  return {
-    content: texts,
-    tiers: contentResult.result?.tiers,
-    loading: contentResult.loading,
-    error: contentResult.error,
-  }
-}
-
-const useAmountToAdd = (tiers: Tier[] | undefined) => {
-  const dollarBalanceString = useSelector(stableTokenBalanceSelector) ?? '0'
-  const dollarBalance = new BigNumber(dollarBalanceString)
-  if (!tiers || tiers.length === 0) {
-    return []
-  }
-  if (dollarBalance.isGreaterThanOrEqualTo(tiers[tiers.length - 1].minBalanceCusd)) {
-    return []
-  }
-  let nextLevel = 1
-  while (dollarBalance.isGreaterThanOrEqualTo(tiers[nextLevel - 1].minBalanceCusd)) {
-    nextLevel++
-  }
-  return [
-    new BigNumber(tiers[nextLevel - 1].minBalanceCusd).minus(dollarBalance).toFixed(0),
-    nextLevel,
-  ]
-}
-
 type Props = StackScreenProps<StackParamList, Screens.ConsumerIncentivesHomeScreen>
 export default function ConsumerIncentivesHomeScreen(props: Props) {
   const { t } = useTranslation(Namespaces.consumerIncentives)
   const userIsVerified = useSelector((state) => state.app.numberVerified)
-  const { content, tiers, loading, error } = useConsumerIncentivesContent()
-  const [amountToAdd, nextLevel] = useAmountToAdd(tiers)
+  const accountAddress = useSelector(accountAddressSelector)
   const insets = useSafeAreaInsets()
-  const dispatch = useDispatch()
 
-  if (!loading && error) {
-    Logger.error(TAG, 'Error while loading remote texts from Firebase', error)
-    dispatch(showError(ErrorMessages.FIREBASE_FETCH_FAILED))
-    navigateBack()
-    return null
-  }
+  const {
+    rewardsABTestThreshold,
+    rewardsAPercent,
+    rewardsBPercent,
+    rewardsStartDate,
+  } = useSelector((state) => state.app)
+  const startDate = new Date(rewardsStartDate).toLocaleDateString(undefined, {
+    month: 'long',
+    day: 'numeric',
+  })
+  const rate =
+    parseInt(accountAddress!, undefined) < parseInt(rewardsABTestThreshold, undefined)
+      ? rewardsAPercent
+      : rewardsBPercent
+  const maxSaving = rate * 10
```

Comment:
I think this would be clearer if we did `const maxSaving = (rate / 100) * MAX_QUALIFIED_BALANCE`, and have `MAX_QUALIFIED_BALANCE` be `1000`. If we move this variable to Firebase then that'd be just fetched from the store instead